src/System.Private.Xml/src/System/Xml/Serialization/ReflectionXmlSerializationReader.cs

```diff
@@ -14,8 +14,15 @@
 using System.Xml.Extensions;
```

Comment:
We shouldn't need `ReflectionXmlSerializationReader` or `ReflectionXmlSerializationWriter`. Can you please open an issue for tracking excluding these files from Sgen. We would also need to address the conditions for using reflection based serialization.